### PR TITLE
URSA-94: Refactor oid references

### PR DESCRIPTION
**Pull request by devin-petersohn**

### CODE REVIEW

File: ursa/graph/graph.py [hunk 1]
```diff
@@ -36,46 +37,47 @@ def insert(self, key, oid, local_edges, foreign_edges, transaction_id):
...
             temp_vertex = temp_vertex.add_local_edges(
                 transaction_id, local_edges)
             temp_vertex = temp_vertex.add_foreign_edges(
                 transaction_id, foreign_edges)
             self.vertices[key].append(temp_vertex)
 
     @ray.method(num_return_vals=0)
-    def update(self, key, node, local_edges, foreign_edges, transaction_id):
-        """Updates the data for a node in the graph.
+    def update(self, key, vertex_data, local_edges, foreign_edges,
```
Comment (madelyn-petersohn) [inline ursa/graph/graph.py]: I don't see an @param  declaration for local_edges.

File: ursa/local_manager.py [hunk 2]
```diff
@@ -235,7 +235,7 @@ def split_graph(self, graph_id):
 
 
 @ray.remote
-def _get_local_edges(edges_oid, buffer_oid):
-    remotes = ray.get(edges_oid)
-    remotes.append(buffer_oid)
+def _get_local_edges(edges, buf):
```
Comment (madelyn-petersohn) [inline ursa/local_manager.py:238]: Are we missing the docstring for this function?

**Review by madelyn-petersohn — CHANGES_REQUESTED** [review]
Just a couple of commenting issues to look at

File: ursa/local_manager.py (hunk 2 above)
Comment (devin-petersohn) [reply ursa/local_manager.py:238]: Yes, I will add

File: ursa/graph/graph.py (hunk 1 above)
Comment (devin-petersohn) [reply ursa/graph/graph.py]: I accidentally deleted it.

**Review by madelyn-petersohn — APPROVED** [review]
Looks great

**Pull request merged**